src/common/bottom_up_merge.cc

```diff
@@ -48,7 +148,7 @@ agd::Status BottomUpMerge::RunMulti(size_t num_threads,
                                     size_t dup_removal_threshold,
                                     AllAllExecutor* executor,
                                     MergeExecutor* merge_executor,
-                                    bool do_allall) {
+                                    bool do_allall,std::vector <std::string>& datasetsFileName) {
```

Comment:
try to stick with the existing naming style `datasetsFileName` --> `datasets_file_name`